eng/source-build-release.yml

```diff
@@ -0,0 +1,121 @@
+trigger: none
+pr: none
+
+pool:
+  name: NetCore1ESPool-Svc-Internal
+  demands: ImageOverride -equals 1es-ubuntu-2004
+
+parameters:
+  - name: sdkVersion
+    displayName: SDK Version
+    type: string
+    default: 6.0.XYY
+  - name: useCustomTag
+    displayName: Use custom tag?
+    type: boolean
+    default: false
+  - name: customTag
+    displayName: Installer custom tag
+    type: string
+    default: v6.0.XYY-source-build
+
+variables:
+  repoId: ''
+  discussionCategoryId: ''
+  announcementOrg: 'dotnet'
+  announcementRepo: 'source-build'
+
+stages:
+
+- stage: Setup
+  jobs:
+  - job: SetupJob
+    displayName: Setup
+    steps:
+    - script: |
+        set -euxo pipefail
+        if [ "${{ parameters.useCustomTag }}" = "True" ] ; then
+          tag=${{ parameters.customTag }}
+          echo "Using custom tag $(tag)"
+        else
+          tag=v${{ parameters.sdkVersion }}
+          echo "Using tag $(tag)"
+        fi
+        echo "##vso[task.setvariable variable=Tag;isOutput=true]${tag}"
+      name: SetTag
+      displayName: Set tag
+    - script: |
+        set -euxo pipefail
+        tag="$(SetTag.Tag)"
+        cd $(Build.SourcesDirectory)
+        git show-ref --tags
+        if git rev-parse "$tag" >/dev/null 2>&1; then
+          echo "Tag $tag exists.";
+        else
+          echo "Tag $tag does not exist."
+          exit 1
+        fi
+      displayName: Ensure tag exists
+
+- stage: ReleaseAnnouncement
+  displayName: Release Announcement
+  dependsOn: Setup
+  jobs:
+  - job: create_release_discussion_announcement
```

Comment:
The '_' naming convention seems inconsistent with the pascal naming used elsewhere.